src/js/player.js

```diff
@@ -1207,6 +1378,8 @@ class Player extends Component {
       // Don't trigger a change event unless it actually changed
       if (this.controls_ !== bool) {
         this.controls_ = bool;
+        this.techCall('setControls', bool);
```

Comment:
Controls are complicated... Here we should check if `usingNativeControls()` returns true before passing the controls setting to the tech. Otherwise we'll end up with both the custom and native controls showing.